statsmodels/tsa/vector_ar/var_model.py

```diff
@@ -392,121 +593,168 @@ def fit(self, maxlags=None, method='ols', ic=None, trend='c',
         method : {'ols'}
```

Comment:
there is currently no choice? 
fit uses `_estimate_var` which doesn't have alternative methods